addons/account/models/account.py

```diff
@@ -705,7 +705,8 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
                 taxes += ret['taxes']
                 continue
 
-            tax_amount = tax._compute_amount(base, price_unit, quantity, product, partner)
+            tax_excluded = tax_included = tax_base = base
```

Comment:
`base` is unpredictably volatile across an unkown series of taxes, what could be the reason to normalize variables on its instantaneous value?